### PR TITLE
change call to get document width and hight

### PR DESCRIPTION
**Pull request by tb102122**
*Description of changes: *
change function call to get document dimensions of PDF

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: overlayer/textractoverlayer/image_tools.py [hunk 1]
```diff
@@ -5,14 +5,14 @@
 import boto3
 
 # Conditionally add /opt to the PYTHON PATH
-if os.getenv('AWS_EXECUTION_ENV') is not None:
```
Comment (schadem) [inline overlayer/textractoverlayer/image_tools.py:8]: Can you leave the "" format? Same applies to the other ' to " changes. Just clutters the commit.

File: overlayer/textractoverlayer/image_tools.py [hunk 2]
```diff
@@ -22,16 +22,18 @@ def get_size_from_filestream(fs, ext) -> DocumentDimensions:
         return DocumentDimensions(doc_width=img.width, doc_height=img.height)
     else:
         input1 = PdfFileReader(fs)
-        pdf = input1.getPage(0).mediaBox
-        return DocumentDimensions(doc_width=pdf.width, doc_height=pdf.height)
+        pdf_page = input1.getPage(0).mediaBox
+        return DocumentDimensions(
+            doc_width=float(pdf_page.getWidth()), doc_height=float(pdf_page.getHeight())
```
Comment (schadem) [inline overlayer/textractoverlayer/image_tools.py:27]: Can you convert to int? DocumentDimensions expect int atm

        return DocumentDimensions(doc_width=int(pdf_page.getWidth()), doc_height=int(pdf_page.getHeight()))

**Review by schadem — CHANGES_REQUESTED** [review]
Thx a lot for the PR. Very minor comments. 
Definitely missing a test here, let me know if I should add one or if you want to add.

**Comment by tb102122** [comment]
> Thx a lot for the PR. Very minor comments. Definitely missing a test here, let me know if I should add one or if you want to add.

It would be great if you can help with the test, I am not sure how you like to test it.

**Pull request merged**